### ISSUE TITLE
Improving geo-search: bounding box

### ISSUE THREAD

**Issue by mlemesle** [meilisearch/meilisearch#2757]
Hey ! I was wondering if this discussion is still up to date https://github.com/meilisearch/product/discussions/431 ?

If so, may I implement this feature ? 🙂

**Comment by curquiza** [comment]
Hello @mlemesle! Thanks a lot for your involvement, this is really appreciated! 😄 🎉 

I asked our PM directly in the discussion! If we want to add it, I will open a dedicated issue with the API and behavior expectations! Indeed we need to do some works on the product side before starting the implementation (e.g. how the API should look like)

Thanks again for working with us!! ❤️ 

Closing this in favor of https://github.com/meilisearch/product/discussions/431#discussioncomment-3586320

**Comment by mlemesle** [comment]
Thanks for allowing me to contribute!
I'll try to follow the discussion and catch the issue when it'll be prioritized then!
Thanks again!
